src/common/grpc-expr/src/error.rs

```diff
@@ -113,6 +116,8 @@ impl ErrorExt for Error {
             Error::ColumnDefaultConstraint { source, .. } => source.status_code(),
             Error::InvalidColumnDef { source, .. } => source.status_code(),
             Error::UnrecognizedTableOption { .. } => StatusCode::InvalidArguments,
+            Error::UnexpectedValuesLength { .. } => StatusCode::Internal,
+            Error::ColumnAlreadyExists { .. } => StatusCode::Internal,
```

Comment:
I think these two errors' status codes should be `StatusCode::InvalidArguments`.